.github/actions/file-diff/index.js

```diff
@@ -52,7 +52,7 @@ async function run() {
 		 * and not just reporting on the overall size of the compiled assets
 		 * @type boolean
 		 */
-		const hasDiff = baseOutput.size > 0;
+		const hasBase = baseOutput.size > 0;
```

Comment:
I renamed this because it isn't so much telling us if the base branch has any diffs as it is telling us if the base branch has any files.